aviary/wren/utils.py

```diff
@@ -163,8 +163,8 @@ def get_aflow_label_from_spglib(
     aflow_label_with_chemsys = get_aflow_label_from_spg_analyzer(spg_analyzer, errors)
 
     # try again with refined structure if it initially fails
-    # NOTE structures with magmoms fail unless all have same magmom
-    if "Invalid" in aflow_label_with_chemsys:
+    # NOTE structures with magmoms fail unless all have same magnetic moment
+    if "invalid" in aflow_label_with_chemsys:
```

Comment:
is it no long capitalized?

Reply:
https://github.com/CompRhys/aviary/blob/f5591923d20abfb57903a23b438cef4cd9d09b91/aviary/wren/utils.py#L142